apidef/oas/middleware_test.go

```diff
@@ -93,3 +93,81 @@ func TestMockResponse(t *testing.T) {
 
 	assert.Equal(t, emptyMockResponse, resultMockResponse)
 }
+
+func TestTransformRequestBody(t *testing.T) {
```

Comment:
Can you please write an empty `TransformRequestBody` test which correctly rebuilds the empty object? We have done similar for all.